### ISSUE TITLE
Default to mounting volumes with nodev, noexec and nosuid options

### ISSUE THREAD

**Issue by gtirloni** [docker/docker#12143]
To prevent code inside a container from creating objects that could be potential security holes on the host.

Might also show a warning if a container is started and the filesystem used to mount a volume does not have those options ("the filesystem used allows device, executable and suid files to be created, please consider using a separate filesystem with nodev,noexec,nosuid options").

**Comment by cpuguy83** [comment]
I think we can provide these as options.
Parsing for this should be available once #5910 is merged.

**Comment by thaJeztah** [comment]
For reference, #5910 is replaced by / carried in #12572

**Comment by thaJeztah** [comment]
Actually, I think this is a duplicate of (or *very* similar to) #7054.

Are you okay with me closing this issue, or do you think this is different enough to keep it open?

**Comment by gtirloni** [comment]
I think it's OK to close this issue, thank you!

**Comment by thaJeztah** [comment]
@gtirloni I'll close this issue, but perhaps you should add a comment to that other issue as well, because this issue has *some* differences; it's about doing this *by default* and more explicitly includes "nodev" and "nosuid".

Ping me anytime if you think it should be re-opened, I'm just trying to keep the discussions in a single place here :)
